docs/src/playground/ExamplePage.tsx

```diff
@@ -1,32 +1,109 @@
-import { Grid } from '@mui/material';
+import { Box, Button, ButtonGroup, Divider, Theme, Typography } from '@mui/material';
+import { Graph, Icon } from '@ui-kit-2022/components';
 
 import AppBar from './components/AppBar';
+import KeyStatistics from './components/KeyStatistics';
 import Sidebar from './components/Sidebar';
+import StockPriceFooter from './components/StockPriceFooter';
 
-export default function ExamplePage() {
+const keyStats = {
+  open: '-',
+  high: '-',
+  low: '-',
+  close: '-',
+  dayRange: '-',
+  yearRange: '-',
+  marketCap: '-',
+  p2eRatio: '-',
+  dividendYield: '-',
+  eps: '-',
+  volume: '-',
+  totalAverageVolume: '-',
+};
+
+const styles = {
+  footer: ({ palette }: Theme) => ({
+    position: 'sticky',
+    bottom: 0,
+    backgroundColor: { light: 'paper.white', dark: 'paper.black' }[palette.mode],
+  }),
+  container: ({ breakpoints }: Theme) => ({
+    display: 'flex',
+    minHeight: '100vh',
+    [breakpoints.down('lg')]: {
+      flexDirection: 'column',
+    },
+  }),
+  sidebar: ({ breakpoints }: Theme) => ({
+    flex: 1,
+    overflow: 'hidden',
+    [breakpoints.up('md')]: {
+      maxWidth: '400px',
+    },
+  }),
+  content: ({ breakpoints }: Theme) => ({
+    display: 'flex',
+    flex: 1,
+    minWidth: 0,
+    [breakpoints.down('md')]: {
+      flexDirection: 'column',
+    },
+  }),
+};
+
+type PartialNumberType = number | undefined;
+
+export interface ExamplePageProps {
+  graphProps: {
+    yLabelStep?: number;
+    xLabelStep?: number;
+    labels: string[];
+    data: PartialNumberType[][];
+    previousData?: number;
+  };
+}
+
+export default function ExamplePage({ graphProps }: ExamplePageProps) {
   return (
-    <Grid container columns={{ xs: 2, sm: 4, md: 8, lg: 12 }}>
-      <Grid item xs={12} md={1}>
+    <Box sx={styles.container}>
+      <Box>
         <AppBar />
-      </Grid>
-      {/* This is just a test of StockerPriceFooter now, later based on the overall layout and the figma design, 
-      it will be modified to fit the whole page  */}
-      {/* <Grid item md={8}>
-        <Box
-          display={'flex'}
-          flexDirection={'column'}
-          height={'100vh'}
-          justifyContent={'flex-end'}
-        >
-          <Box></Box>
+      </Box>
+      <Box sx={styles.content}>
+        <Box flex="2" display="flex" flexDirection="column" px={4} sx={{ minWidth: 0 }}>
+          <Box mt={6}>
+            <Typography variant="h3" sx={{ p: 2 }}>
```

Comment:
A `Ticker Search` element should be used instead of a Typography element here.  When the user interacts with this element they should be able to change the text to perform another stock / currency search.

The demo RA app seems to be working again, so you can see an example of that [interaction here](https://demo-reactive-analytics.adaptivecluster.com/stock/IBM).